sidebars.js

```diff
@@ -1,468 +1,120 @@
 /**
- * Creating a sidebar enables you to:
- - create an ordered group of docs
- - render a sidebar for each doc of that group
- - provide next/previous navigation
-
- The sidebars can be generated from the filesystem, or explicitly defined here.
-
- Create as many sidebars as you want.
+ * Revised sidebar configuration for the MultiversX documentation,
+ * designed to help new developers navigate the learning resources effectively.
  */
 
 // @ts-check
 
 /** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
 const sidebars = {
-  // By default, Docusaurus generates a sidebar from the docs folder structure
-  // tutorialSidebar: [{ type: "autogenerated", dirName: "." }],
-
-  // But you can create a sidebar manually
-  /*
   tutorialSidebar: [
-    'intro',
-    'hello',
     {
       type: 'category',
-      label: 'Tutorial',
-      items: ['tutorial-basics/create-a-document'],
+      label: 'Getting Started',
+      items: [
+        'welcome/welcome-to-multiversx',
+        'technology/architecture-overview',
+        'technology/glossary',
+        'technology/entities',
+        'technology/chronology',
+        'setup/installing-mxpy',
+        'setup/configuring-mxpy',
+        'setup/cli-overview',
+        'setup/build-reference',
+        'setup/configuration',
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Core Blockchain Concepts',
+      items: [
+        'blockchain-basics/secure-proof-of-stake',
+        'blockchain-basics/adaptive-state-sharding',
+        'smart-contracts/smart-contracts-overview',
+        'smart-contracts/multiversx-smart-contracts',
+        'smart-contracts/smart-contract-api-functions',
+        'smart-contracts/smart-contract-annotations',
```

Comment:
Smart contract development is not a core concept. Especially none of the mx sc scpecifics. It's not even a Rust-only thing anymore.

So we can have something on the fundamentals here, but nothing concrete, certainly no smart contract annotations/api/etc.